app-crypt/rhash/rhash-1.3.6-r1.ebuild

```diff
@@ -59,8 +60,9 @@ multilib_src_configure() {
 multilib_src_install() {
 	# -j1 needed due to race condition.
 	emake DESTDIR="${D}" -j1 \
-		  install{,-lib-so-link,-pkg-config} \
-		  $(use nls && echo install-gmo)
+		  install{,-pkg-config} \
+		  $(use nls && echo install-gmo) \
+		  $(use kernel_linux && echo install-lib-so-link)
```

Comment:
Linux and Windows aren't the only kernels we're dealing with. This should be:

```sh
$(use kernel_Winnt || echo install-lib-so-link)
```

Reply:
@chewi well, unless I'm mistaken linux is the only OS which uses a '*.so' extension, but I'll go ahead and change it here :)

Reply:
Most of the BSDs (not Darwin) use `.so` too.